DevTools/HackStudio/Debugger/BacktraceModel.h

```diff
@@ -47,12 +47,15 @@ class BacktraceModel final : public GUI::Model {
     virtual void update() override {}
     virtual GUI::ModelIndex index(int row, int column = 0, const GUI::ModelIndex& = GUI::ModelIndex()) const override { return create_index(row, column, &m_frames.at(row)); }
 
-private:
     struct FrameInfo {
         String function_name;
-        u32 address_in_frame;
+        u32 instruction_address;
+        u32 frame_base;
     };
 
+    const Vector<FrameInfo> frames() const { return m_frames; }
```

Comment:
This will create a copy of the vector every times it's called. Let's use a const reference:
```suggestion
    const Vector<FrameInfo>& frames() const { return m_frames; }
```